assets/js/main.js

```diff
@@ -17,6 +17,74 @@ let peerjsInitalized = false;
 // To remember last pressed button
 let lastPressedStart = false;
 
+// Save code to file
+window.saveCodeToFile = function saveCodeToFile(){
```

Comment:
@hanneseilers could you add spaces before the "{", so the code style is consistent